pkg/v1/config/clientconfig.go

```diff
@@ -57,6 +57,9 @@ const (
 	// TKR version v1alpha3 feature flag determines whether to use Tanzu Kubernetes Release API version v1alpha3. Setting
 	// feature flag to true will allow to use the TKR version v1alpha3; false allows to use legacy TKR version v1alpha1
 	FeatureFlagTKRVersionV1Alpha3 = "features.global.tkr-version-v1alpha3-beta"
+	// Package Plugin Kctrl Command Tree determines whether to use the command tree from kctrl. Setting feature flag to
+	// true will allow to use the package command tree from kctrl for package plugin
+	FeatureFlagPackagePluginKctrlCommandTree = "features.package.kctrl-package-command-tree"
```

Comment:
Is there any convention on deciding the feature flag name? I am open to suggestions on this.